images/scripts/rhel.setup

```diff
@@ -103,7 +103,7 @@ EOF
 
     elif [ "${IMAGE#rhel-8*}" != "$IMAGE" ]; then
         case "$IMAGE" in
-            rhel-8-3) REPO="rhel-8/nightly/RHEL-8/latest-RHEL-8.3" ;;
+            rhel-8-3) REPO="rhel-8/nightly/updates/RHEL-8/latest-RHEL-8.3.1" ;;
```

Comment:
While this change makes sense, let's leave that to PR #1339

Reply:
oh, damn, I had some local changes I committed by accident.